ivadomed/main.py

```diff
@@ -292,9 +292,18 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
         bids_ds = bids.BIDS(context["loader_parameters"]["bids_path"])
         subj_lst = bids_ds.participants.content['participant_id'].tolist()
         bids_subjects = [s for s in bids_ds.get_subjects() if s.record["subject_id"] in subj_lst]
+
+        # Add postprocessing to packaged model
+        path_model = os.path.join(context['log_directory'], context['model_name'])
+        path_model_config = os.path.join(path_model, context['model_name'] + ".json")
+        model_config = imed_config_manager.load_json(path_model_config)
+        model_config['postprocessing'] = context['postprocessing']
+        with open(path_model_config, 'w') as fp:
+            json.dump(model_config, fp, indent=4)
```

Comment:
So if I understand correctly here: it allows the user to `segment` with postpro params that are potentially different from the ones defined in the initial config file (ie the one used to train the model). Is it correct?

Reply:
Yes, exact: the postprocessing applied is the one in the current config file. During training, postprocessing steps are not applied, so it is likely at inference the user might want to modify these steps.  Would we want to resave the config file in the packaged model to update de postprocessing steps?

Reply:
> Would we want to resave the config file in the packaged model to update de postprocessing steps?

Do you mean: update the config file in the packaged model with the postprocessing params input by the user? If so: no I don't think so: the user may have different needs than the packaged model and it could create confusion. I would leave it as you implemented. All good to me!